ntp-daemon/src/config/peer.rs

```diff
@@ -1,52 +1,72 @@
 use std::{
     fmt,
     net::SocketAddr,
+    ops::Deref,
     path::PathBuf,
     sync::{Arc, Mutex},
 };
 
 use rustls::Certificate;
-use serde::{
-    de::{self, MapAccess, Visitor},
-    Deserialize, Deserializer,
-};
+use serde::{de, Deserialize, Deserializer};
 
 use crate::keyexchange::certificates_from_file;
 
-#[derive(Deserialize, Debug, PartialEq, Eq, Clone, Copy, Default)]
-pub enum PeerHostMode {
-    #[serde(alias = "server")]
-    #[default]
-    Server,
-    #[serde(alias = "nts-server")]
-    NtsServer,
-    #[serde(alias = "pool")]
-    Pool,
-}
-
 #[derive(Deserialize, Debug, PartialEq, Eq, Clone)]
 #[serde(rename_all = "kebab-case", deny_unknown_fields)]
 pub struct StandardPeerConfig {
-    pub addr: NormalizedAddress,
+    #[serde(rename = "address")]
+    pub addr: NtpAddress,
 }
 
-#[derive(Debug, PartialEq, Eq, Clone)]
+#[derive(Debug, Deserialize, PartialEq, Eq, Clone)]
 pub struct NtsPeerConfig {
```

Comment:
```suggestion
#[derive(Debug, Deserialize, PartialEq, Eq, Clone)]
#[serde(deny_unknown_fields)]
pub struct NtsPeerConfig {
```
To match current behaviour.